tests/unit/importer.js

```diff
@@ -168,5 +171,47 @@ describe('importer', function() {
         });
     });
 
+    describe('different scenarios for get sequence', function() {
+      it('finds sequence for given source', function() {
+        sinon.stub(db, 'one').returns(successfulPromise(STORED_SEQ));
+
+        return importer(db, couchdb).getSeq('localhost:5984/simon')
+          .then(function(seq) {
+            expect(seq).to.equal(STORED_SEQ.seq);
```

Comment:
Please don't change the style of assert on half the file, it's really confusing and inconsistent.

This file as it exists uses the should should style. I picked this because it seems to be the dominant style in the rest of medic's code. You should almost certainly follow suite.

However, if you feel very strongly that the expect style is better, you can change it, but you need to change the whole file (and ideally all of couch2pg), not just the new parts. Imagine in a few more months with a few more contributers, and the tests are a muddle of a bunch of different styles, it would not be great :-)

Reply:
To be additionally clear, drop the `expect` pull in at the top, and re-write this (and all the others) like so:
```
// expect(seq).to.equal(STORED_SEQ.seq);
seq.should.equal(STORED_SEQ.seq);
```